File: Beecrowd/C/bee-1101.c
#include <stdio.h>

int main() {
    int m, n, smallest, biggest;
    while(1){
        scanf("%d %d", &m, &n);
        if(m <= 0 || n <= 0){
            break;
        }
        if(m >= n){
            biggest = m;
            smallest = n;
        } 
        else if(m <= n){
            biggest = n;
            smallest = m;
        }
        int total = 0;
        for(int i = smallest; i <= biggest; i++){
            total += i;
        }
        for(int i = smallest; i <= biggest; i++){
            printf("%d ", i);
        }
        printf("Sum=%d\n", total);
    }

    return 0;
}

// 1101 - Sequence of Numbers and Sum
// https://www.beecrowd.com.br/judge/en/problems/view/1101

File: Python/Beecrowd/bee-1101.py
while True:
    m, n = map(int, input().split())
    if m <= 0 or n <= 0:
        break
    smallest = min(m, n)
    biggest = max(m, n)
    total = 0

    for i in range(smallest, biggest+1):
        total += i
        print(i, end=' ')
    print(f'Sum={total}')

# 1101 - Sequence of Numbers and Sum
# https://www.beecrowd.com.br/judge/en/problems/view/1101